.circleci/config.yml

```diff
@@ -10,8 +10,6 @@ jobs:
     working_directory: ~/repo
```

Comment:
If we use `shell.nix` to build dev environment locally and production environment on Heroku, let's also use it for CI. So we don't break it by mistake.